src/sds.c

```diff
@@ -1470,27 +1472,27 @@ int sdsTest(int argc, char **argv, int flags) {
         x = sdsResize(x, 200, 1);
         test_cond("sdsrezie() expand len", sdslen(x) == 40);
         test_cond("sdsrezie() expand strlen", strlen(x) == 40);
-        test_cond("sdsrezie() expand alloc", sdsalloc(x) == 200);
+        test_cond("sdsrezie() expand alloc", sdsalloc(x) >= 200);
```

Comment:
These need to get ported, but can we also add some reasonable upper bound like 
```suggestion
        test_cond("sdsrezie() expand alloc", sdsalloc(x) >= 200 && sdsalloc(x) < 400);
```
Also document at least once why we are doing the bounds checking.